scripts/build/linux-cross

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Build static linux binary for multiple architectures
+#
+
+set -eu -o pipefail
+
+source ./scripts/build/.variables
+
+CROSS_OSARCH="linux/amd64 linux/arm"
+
+# Compile is broken
+# linux/ppc64le
+
+# Not yet supported by gox
+# linux/s390x
+
+gox -output build/docker-{{.OS}}-{{.Arch}} \
```

Comment:
Can we print something here (and the others) telling what we're building? The compilation can take quite a time sometimes, and all we get is the sha output of the container being used (and this one is there without context). It's kind of weird to see a blank terminal and not knowing if the program is stuck or doing something.

A simple `Building {x}...` would be nice :)